### PR TITLE
1. fixed none object reffernce issue.

### PR DESCRIPTION
**Pull request by azraelrabbit**
2. fixed non-english culture the static resource last-modified datetime formatting issue.

**Changes**

**Issues**

### CODE REVIEW

File: Emby.Server.Implementations/Services/ServiceHandler.cs [hunk 1]
```diff
@@ -26,7 +26,16 @@ protected static Task<object> CreateContentTypeRequest(HttpListenerHost host, IR
             if (!string.IsNullOrEmpty(contentType) && httpReq.ContentLength > 0)
             {
                 var deserializer = RequestHelper.GetRequestReader(host, contentType);
-                return deserializer?.Invoke(requestType, httpReq.InputStream);
+                var invokedDeserializer = deserializer?.Invoke(requestType, httpReq.InputStream);
```
Comment (cvium) [inline Emby.Server.Implementations/Services/ServiceHandler.cs]: There is already a PR for this.

File: Jellyfin.Server/Program.cs [hunk 2]
```diff
@@ -38,6 +40,11 @@ public static class Program
 
         public static async Task Main(string[] args)
         {
+            // for encoding issue  to register codepages by azraelrabbit
```
Comment (cvium) [inline Jellyfin.Server/Program.cs]: Unnecessary comment tbh

File: Jellyfin.Server/Program.cs [hunk 3]
```diff
@@ -38,6 +40,11 @@ public static class Program
...
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.InvariantCulture;
```
Comment (cvium) [inline Jellyfin.Server/Program.cs]: I'm not sure how I feel about this change. What are the consequences exactly and why is it needed?

Comment (azraelrabbit) [reply Jellyfin.Server/Program.cs]: i running jellyfin in ubuntu 18.04.2 x64 ,and locale is  zh-cn.UTF-8, timezone is asia/shanghai.
that cause the DateTime.ToString() result like this: " 2019-04-25 下午9:30" , 
all the static file like *.js  *.ico request from webbrowser will got exception : 

`System.InvalidOperationException: Invalid non-ASCII or control character in header: 0x4E0B
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpHeaders.ThrowInvalidHeaderCharacter(Char ch)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpHeaders.ValidateHeaderValueCharacters(StringValues& headerValues)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpResponseHeaders.AddValueFast(String key, StringValues& value)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpHeaders.System.Collections.Generic.IDictionary<System.String,Microsoft.Extensions.Primitives.StringValues>.Add(String key, StringValues value)
   at Emby.Server.Implementations.Services.ResponseHelper.WriteToResponse(IResponse response, IRequest request, Object result, CancellationToken cancellationToken)
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost appHost, IRequest httpReq, IResponse httpRes, ILogger logger, String operationName, CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
`

Comment (azraelrabbit) [reply Jellyfin.Server/Program.cs]: and releated  issue is :  #1234

File: Emby.Server.Implementations/Services/ServiceHandler.cs (hunk 1 above)
Comment (azraelrabbit) [reply Emby.Server.Implementations/Services/ServiceHandler.cs]: ok

**Comment by azraelrabbit** [comment]
There is already a PR, and  my pr need modified. 
close this pr and  i'll pull again for  just necessary code.

**Pull request closed without merging**